go.mod

```diff
@@ -1,6 +1,6 @@
 module knative.dev/eventing
 
-go 1.22.0
+go 1.23
```

Comment:
@creydr not sure if it's fixed in `cachi2`, but no patch version failed on fetching deps or something.... needs to be `1.23.0`

Reply:
:roll_eyes: 
updated